e2e/cypress/integration/archived_channel/archive_channel_post_spec.js

```diff
@@ -47,4 +57,71 @@ describe('Archived channels', () => {
         // * RHS text box should not be visible
         cy.get('#reply_textbox').should('not.be.visible');
     });
+
+    it.only('MM-T1722 Can click reply arrow on a post from archived channel, from saved posts list', () => {
+        let testArchivedChannel;
+        let postId;
+        let permalink;
+
+        cy.apiAdminLogin();
+
+        // # Create a channel that will be archived
+        cy.apiCreateChannel(testTeam.id, 'archived-channel', 'Archived Channel').then(({channel}) => {
+            testArchivedChannel = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${testArchivedChannel.name}`);
+
+            // # Post message
+            cy.postMessage('Test');
+
+            // # Create permalink to post
+            cy.getLastPostId().then((id) => {
+                postId = id;
```

Comment:
You may use the `id` as `postId` directly.
```suggestion
            cy.getLastPostId().then((postId) => {
```